kythe/rust/indexer/rustfmt.toml

```diff
@@ -0,0 +1,17 @@
+# Copyright 2020 The Kythe Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+format_code_in_doc_comments = true
+tab_spaces = 2
+wrap_comments = true
```

Comment:
This differs from the google3 consensus (go/rust-style). Any reasons for this?

Reply:
Ah, I didn't know about this! I'll update

Reply:
I notice that the Fuchsia format config doesn't wrap comments. Wouldn't it be better to at least enforce comments to have the same column width as code?

Reply:
I think I'd prefer to see wrapped comments unless this is strongly discouraged by Rust idiom.

Reply:
Keeping wrap_comments but changing the other options to match Fuchsia's